frm/four_roll_mill.cpp

```diff
@@ -156,7 +154,11 @@ main(int argc, char* argv[])
                                              grid_geometry,
                                              adv_diff_integrator,
                                              visit_data_writer);
-            cf_un_forcing->setSigmaScaleFcn(stress_scale);
```

Comment:
Since this line was removed, where does this scaling take place in the new changes? I was unable to find an equivalent change elsewhere and I couldn't find this function in the IBAMR repo. Just wanted to see what it's supposed to do.

Reply:
`CFMultiphaseOldroydB` multiplies by the volume fraction, see https://github.com/abarret/multiphase-stokes/blob/6e7777aec893bb295a6246580612c6b712e95100/src/CFMultiphaseOldroydB.cpp#L58-L122

The `CFINSForcing` now takes in a `CFStrategy` implementation `CFMultiphaseOldroydB` that computes how the conformation tensor relaxes and how to convert from conformation tensor to stress.